src/main/java/randoop/condition/OperationConditions.java

```diff
@@ -0,0 +1,175 @@
+package randoop.condition;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Represents the collection of all conditions on an operation including pre-, post- and
```

Comment:
This comment is not correct:  it isn't just a collection of conditions, but other data structures.
This means that the name of the class needs to be changed (it's misleading to name it Operation**Conditions**).
Also incorrect is the use of "conditions" in the Javadoc `returnConditions`, `throwsConditions`, and `parentList` (and the former two fields should probably be renamed).
The `ConditionPair` class name is not informative; it serves little purpose, so replace it by two classes with descriptive names.